orange-stock-ios/orange-stock-ios/Controllers/FavoriteStockListViewController.swift

```diff
@@ -69,40 +87,53 @@ extension FavoriteStockListViewController {
          ?? UITableViewCell()
     }
 }
+ 
+// MARK: - Layout
 
-// MARK: Enum Attributes
-
-private extension FavoriteStockListViewController {
-    /// navigation
-    enum Attributes {
-        static let title = "관심 주식 목록"
-        static let searchImage = "magnifyingglass"
-    }
+extension FavoriteStockListViewController: LayoutProtocol {
     
-    /// Cell Identifier
-    enum CellID {
-        static let stockListTableViewCell = "FavoriteStockListTableViewCell"
-        static let additionTableViewCell = "FavoriteStockAdditionTableViewCell"
-        static let stockListHeaderView = "FavoriteStockListHeaderView"
+    func layout() {
+        setNavigation()
+        attributes()
+        register()
     }
-}
- 
-// MARK: Layout
-
-private extension FavoriteStockListViewController {
-
+    
+    // MARK: Navigation
+    
     func setNavigation() {
         self.navigationController?.navigationBar.prefersLargeTitles = true
         title = Attributes.title
-        
+        self.navigationItem.rightBarButtonItems = makeRightBarButtonItems()
+    }
+    
+    func makeRightBarButtonItems() -> [UIBarButtonItem] {
+        let searchBarButton = makeSearchBarButtonItem()
+        let settingBarButton = makeSettingBarButtonItem()
+        return [settingBarButton, searchBarButton]
```

Comment:
이건 그냥 인라인 코드로 처리해도 충분하지 않을까요? 

```suggestion

        return [makeSearchBarButtonItem(), makeSettingBarButtonItem()]
```